api_tests/e2e/rfc003/eth_btc/test.js

```diff
@@ -302,32 +201,34 @@ describe("RFC003: Ether for Bitcoin", () => {
             );
         res.should.have.status(200);
         alice_redeem_action = res.body;
-
-        logger.info(
-            "Alice retrieved the following redeem parameters",
-            alice_redeem_action
-        );
     });
 
-    let alice_btc_balance_before;
-
     it("[Alice] Can execute the redeem action", async function() {
         alice_redeem_action.payload.should.include.all.keys("hex", "network");
-        alice_btc_balance_before = await bitcoin.btc_balance(
-            alice_final_address
-        );
         await alice.do(alice_redeem_action);
         await bitcoin.btc_generate();
     });
 
     it("[Alice] Should have received the beta asset after the redeem", async function() {
-        let alice_btc_balance_after = await bitcoin.btc_balance(
-            alice_final_address
+        this.timeout(10000);
+        let body = await alice.poll_comit_node_until(
+            chai,
+            alice_swap_href,
+            body => body.state.beta_ledger.status == "Redeemed"
+        );
+        let alice_redeem_txid = body.state.beta_ledger.redeem_tx;
+        let alice_redeem_tx = await bitcoin.getRawTransaction(
+            alice_redeem_txid
         );
 
-        const alice_btc_balance_expected =
-            alice_btc_balance_before + beta_asset_amount - beta_max_fee;
-        alice_btc_balance_after.should.be.at.least(alice_btc_balance_expected);
+        let receive_address = alice_redeem_tx.vout[0].scriptPubKey.addresses[0];
+        receive_address.should.be.equal(alice_final_address);
```

Comment:
:x: Instead of _verifying_ this, I would _search_ for the UTXO with the receive address and compare the value against the expected one.

You don't want to test that the COMIT node gives you the right TX (at least not primarily). You want to verify that this TX pays Alice enough money. If you can't find a UTXO that pays Alice money, it is not enough.

There also seems to be some repetition here that could be extracted into a fn like `get_satoshi_transferred_to(tx_id, address)`.